### PR TITLE
Ignore Dependabot Upgrades for DotNet 7 Assemblies

### PR DESCRIPTION
**Pull request by bgfoshay**
## What’s changing?

Update Dependabot file to ignore .Net 7 dependency bumps

## Why?

Not planning to upgrade to .Net 7 at this time. 

## This PR has:

- [x] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -160,68 +593,128 @@ updates:
...
+    - dependency-name: "System.Configuration.ConfigurationManager"
+      versions: ["7.x"]      
+    - dependency-name: "System.Text.Json"      
+      versions: ["7.x"]   
 - package-ecosystem: nuget
   directory: "/participants/tests/Piipan.Participants.Core.Tests"
   schedule:
     interval: daily
     time: "09:00"
-- package-ecosystem: nuget
```
Comment (dborgen2) [inline .github/dependabot.yml:183]: Question: Was this intentionally removed?

**Review by ryangeorgeharvey — COMMENTED** [review]
Same question as Dan, why was some of the directory/interval sections removed?

File: .github/dependabot.yml (hunk 1 above)
Comment (bgfoshay) [reply .github/dependabot.yml:183]: Good catch. Not sure how I messed that up. I put them back in.

**Review by dborgen2 — APPROVED** [review]
👍

**Review by ryangeorgeharvey — APPROVED** [review]
Hard to know for sure if this will cover everything before it gets merged, but LGTM.

**Pull request merged**